src/labels/CustomLabels.labels

```diff
@@ -2103,6 +2103,105 @@
         <shortDescription>Title of the Schedule Visualizer Component</shortDescription>
         <value>Upcoming Installments</value>
     </labels>
+    <labels>
+        <fullName>RD2_StatusMappingColumnState</fullName>
+        <categories>Settings</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Mapping datatable State field label</shortDescription>
+        <value>State</value>
+    </labels>
+    <labels>
+        <fullName>RD2_StatusMappingColumnStatus</fullName>
+        <categories>Settings</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Mapping datatable Status field API name</shortDescription>
+        <value>API Name</value>
+    </labels>
+    <labels>
+        <fullName>RD2_StatusMappingColumnStatusLabel</fullName>
+        <categories>Settings</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Mapping datatable Status field label</shortDescription>
+        <value>Status</value>
```

Comment:
Rather than use a new custom label for this, do you think it would be better to use the label of the `npe03__Recurring_Donation__c.Status__c` field?